### PR TITLE
1.6.7

### PR DESCRIPTION
**Pull request by SC426**
[Upstream](https://github.com/microsoft/debugpy/tree/v1.6.7)
[Changelog](https://github.com/microsoft/debugpy/releases)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
-  sha256: d2b09e91fbd1efa4f4fda121d49af89501beda50c18ed7499712c71a4bf3452e
+  sha256: c4c2f0810fa25323abfdfa36cbbbb24e5c3b1a42cb762782de64439c575d67f2
 
 build:
   number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: `- python                                 # [build_platform != target_platform]` can be removed from `requirements/build` as we do not use cross-compilation

**Review by skupr-anaconda — APPROVED** [review]
LGTM but cross-compilation stuff

**Review by akabanovs — APPROVED** [review]
LGTM once @skupr-anaconda  comment has been addressed.

File: recipe/meta.yaml (hunk 1 above)
Comment (SC426) [reply recipe/meta.yaml:13]: It's been removed

**Pull request merged**